megamek/src/megamek/server/GameManager.java

```diff
@@ -19348,7 +19348,9 @@ void checkForPSRFromDamage() {
                         getGame().addControlRoll(damPRD);
                     } else {
                         // was the damage threshold exceeded this round?
-                        if (((IAero) entity).wasCritThresh()) {
+                        // Note errata: https://bg.battletech.com/forums/index.php?topic=72983.msg2024516#msg2024516
+                        if ((((IAero) entity).wasCritThresh())
```

Comment:
I'm setting breakpoints on this method but they never seem to be getting hit--not sure what's going on there but I thought this method should be called every round at least.

Reply:
Added a logging statement and it is actually getting called, so just a problem with my local environment I guess.

Reply:
IDEA will sometimes skip breakpoints if it decides they will always run, or never run.  I would try putting a conditional breakpoint inside `wasCritThresh()`, and have it only fire after passing the breakpoint at 19351.
This can be done from the "more" page for the individual breakpoint, or by opening the breakpoints dialog.